src/components/Accordion/index.tsx

```diff
@@ -82,13 +82,15 @@ const AccordionToggle = styled.span`
   }
 `;
 const AnimatedAccordion = styled.div`
-  max-height: 0;
   /* This is used to calculate a fixed height for the Accordion since height
-     transitions require a fixed height for their end height */
-  max-height: ${(props: { hide: boolean }): string => !props.hide && "6000px"};
-  overflow-y: ${(props: { hide: boolean }): string =>
-    props.hide ? "hidden" : "visible"};
-  transition: max-height 0.3s ease-in-out;
+      transitions require a fixed height for their end height */
+  max-height: ${(props: { hide: boolean }): string =>
+    props.hide ? "0px" : "9999px"};
+  overflow-y: ${(props: { hide: boolean }): string => props.hide && "hidden"};
+  transition: ${(props: { hide: boolean }): string =>
+    props.hide
+      ? "max-height 0.3s cubic-bezier(0, 1, 0, 1)"
+      : "max-height 0.6s ease-in-out"};
```

Comment:
I tinkered with the animation a little bit to make it smoother when collapsing. `ease-in-out` is equivalent to `cubic-bezier(0.42, 0.0, 0.58, 1.0)` so literally it's just a matter of fiddling with the numbers 🤠

Before:

https://user-images.githubusercontent.com/47064971/161283643-7e4382ce-8347-4ba8-9e60-c0ab45e4f71a.mov

After:

https://user-images.githubusercontent.com/47064971/161283692-837236f7-2337-4f6b-aba2-8bdddff04cf7.mov

Reply:
😍